apps/web/components/v2/eventtype/EventTypeSingleLayout.tsx

```diff
@@ -141,7 +141,7 @@ function EventTypeSingleLayout({
   useEffect(() => {
     // Default to the first in the list
     if (!router.query.tabName) {
-      router.push({
+      router.replace({
```

Comment:
why does the /v2/ redirect appear in the first place, anyway? @zomars https://github.com/calcom/cal.com/issues/4579

Reply:
i'll need to wait for @zomars for this to be merged

Reply:
This code got deprecated in 588748d